docs/testing/README.md

```diff
@@ -0,0 +1,123 @@
+# Local TXTDirect Configuration
+## Linux
+* Install CoreDNS
+    ```
+    go get github.com/coredns/coredns
+    ```
+* Add testing domain to /etc/hosts e.g. 127.0.0.1 example.test
+* Create CoreDNS config file named 'Corefile'
+    ```
+    .:5353 {
+      file /path/to/example.test example.test
+      forward . 8.8.8.8
+      errors stdout
+      log
+    }
+    ```
+* Create DNS record file named `example.test`
+    ```
+    @                  3600 IN SOA      ns.example.com (
+                                          2010101010   ; serial
+                                          5m           ; refresh
+                                          5m           ; retry
+                                          1w           ; expire
+                                          12h    )     ; minimum
+    @                  86400 IN NS      ns.example.com.
+    @                  86400 IN NS      ns.example.com.
+
+    @                     60 IN A       127.0.0.1
+    _redirect             60 IN TXT     "v=txtv0;to=full-wildcard.worked.example.test;root=http//root.worked.example.test;type=path;code=302"
+    _redirect.two.one     60 IN TXT     "v=txtv0;to=two-one.worked.example.test;type=host;code=302"
+    redirect..one         60 IN TXT     "v=txtv0;to=two-wildcard.worked.example.test;type=host;code=302"
+    redirect.._           60 IN TXT     "v=txtv0;to=two-one-wildcard.worked.example.test;type=host;code=302"
+    _redirect.nohost      60 IN TXT     "v=txtv0;to=nohost.worked.example.test;code=302"
+    _redirect.host        60 IN TXT     "v=txtv0;host.worked.example.test;type=host;code=302"
+    ```
+* Create a caddyfile named `caddy.test`
+    ```
+    example.test:8080 {
+        tls off
+        txtdirect {
+            enable path host
+            resolver 127.0.0.1:5353
+            logfile stdout
+        }
+        log / stdout "{remote} - [{when_iso}] \"{method} {uri} {proto}\" {status} {size} {latency}"
+        errors stdout
+    }
+    ```
+* Navigate to the directory in terminal where the Corefile was created and start CoreDNS with the Corefile
+    ```
+    coredns -conf Corefile
+    ```
+*  Navigate to the local TXTDirect repository directory in terminal and rebuild TXTDirect
+    ```
+    make build
+    ```
+    *Note: if the above command does not pull in all the changes and create a TXTDirect binary file, use `make travis-build` instead.
```

Comment:
Good for now, but we should probably fix the difference in local and travis builds in the long run.